src/Microsoft.ApplicationInsights.AspNet/ContextInitializers/DomainNameRoleInstanceContextInitializer.cs

```diff
@@ -35,17 +35,16 @@ public void Initialize(TelemetryContext context)
 
         private string GetMachineName()
         {
-            string domainName = IPGlobalProperties.GetIPGlobalProperties().DomainName;
             string hostName = Dns.GetHostName();
-
+            
+#if !dnxcore50
+            string domainName = IPGlobalProperties.GetIPGlobalProperties().DomainName;
```

Comment:
Because in dnxcore we will not have domain in the machine name and I did not find a way to get it fast enough. I will add an issue for that.